Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,36 +14,21 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             String[] expressions = equation.Split(new Char[] { '*', '=' });        //here we split the equation into three parts
             int[] initialize = new int[3];                                         //here we made an array for basically storing at which part is an '?'
             for (int i = 0; i < initialize.Length; i++)
             {
                 initialize[i] = -1;                                                 //initially we initialzie the array by -1 
             }
-            int k = 0;
-            for (int i = 0; i < expressions.Length; i++)
-            {
-                String expression = expressions[i];
-                int flag = 0;
-
-                if (expression.Contains('?'))
-                {                                                   //if the string doesn't contains '?' than we convert the string into integer
-                    flag = 1;                                       // and stores it in array
-                    initialize[k++] = -1;
-                }
-                if (flag == 0)
-                {
-                    initialize[k++] = Int32.Parse(expression);            
-                }                                                   
-            }
+            initialize = checkExpression(initialize,expressions);

```

Comment:
As per naming convention that we follow, All methods and classes name follow pascal case. 
For variables in methods we use camel case e.g myVariable.
For private variables we use '_' followed by camel case, e.g. _myVariable.
For public variables we use pascal case.